app/src/store/store.ts

```diff
@@ -87,46 +41,31 @@ const store = new Vuex.Store({
     }
   },
   actions: {
-    initializeSearchProposals({ commit }, proposals: Tag[]): void {
-      commit('initializeSearchProposals', proposals);
-    },
+
     findPosts({ commit, state }): void {
-      const location = LocationService.findByTitle(state.selectedLocation);
-      const searchValues = state.searchValues;
-      const radius = state.radiusSearchValue;
+      const location = LocationService.findByTitle(state.locationSearchModule.selectedLocation);
+      const searchValues = state.textSearchModule.searchValues;
+      const radius = state.locationSearchModule.selectedRadius;
       DataService.findBySelection({
         searchValues,
         location,
         radius
       }).then((result) => commit('setPosts', result));
     },
-    addSearchValue({ commit, dispatch }, searchValue): void {
-      commit('addSearchValue', searchValue);
-      dispatch('updateURIFromState');
-      dispatch('findPosts');
-    },
-    addSearchValues({ commit, dispatch }, searchValues: string[]): void {
-      searchValues.forEach((tag) => commit('addSearchValue', tag));
-    },
     setResultPage({ commit }, value: number): void {
       commit('setPage', value);
     },
-    removeSearchValue({ commit, dispatch }, value): void {
-      commit('removeSearchValue', value);
-      dispatch('updateURIFromState');
-      dispatch('findPosts');
-    },
     hydrateStateFromURIParams({ commit, dispatch }, queryParams): void {
       // Clear previous search parameters. The URI is our single source of truth!
       commit('clearSearchParams');
       if ('q' in queryParams) {
-        dispatch('addSearchValues', queryParams.q.split(','));
+        dispatch('textSearchModule/addSearchValues', queryParams.q.split(','));
```

Comment:
Just a thought on future maintainability:

We could use a dispatch table (imported from a separate file) here, that loops over the queryParams. 
By doing this, we would move the "place we need to change" for queryParam logic into its own file and have a higher level of abstraction that prevents us from adding / removing conditionals.

Reply:
I would gladly accept a PR for this but don´t think the has priority.